dicoogle/src/main/resources/webapp/js/app.js

```diff
@@ -64,33 +65,23 @@ class App extends React.Component {
 		});
 	}
 
+	componentWillMount() {
```

Comment:
There are a few mixtures of tabs and spaces in the code. Can you do a quick conversion to spaces (only on your changes)?